vmm/src/lib.rs

```diff
@@ -1810,12 +1763,12 @@ mod tests {
         }
     }
 
-    impl KvmContext {
-        pub fn new() -> Result<Self> {
-            let kvm = Kvm::new().map_err(Error::Kvm)?;
-            Self::with_kvm(kvm)
-        }
-    }
+    //impl KvmContext {
```

Comment:
This looks like a leftover, please remove

Reply:
Indeed a leftover, removed it.